### PR TITLE
Add analysis types and names list

### PR DESCRIPTION
**Pull request by KapRom**
- added analysis types and names lists to the side-nav,
- small layout changes

### CODE REVIEW

File: src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.html [hunk 1]
```diff
@@ -0,0 +1,24 @@
+<mat-accordion [displayMode]="flat">
+<mat-expansion-panel *ngFor="let type of types, let i = index"
+                     (opened)="panelOpenState[i] = true"
+                     (closed)="panelOpenState[i] = false">
```
Comment (Sebaxp95) [inline src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.html]: Move it to function

**Comment by gmrukwa** [comment]
[Create generic analyses list panel](https://trello.com/c/0D5nZdW7/416-create-generic-analyses-list-panel)

File: src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.css [hunk 2]
```diff
@@ -0,0 +1,14 @@
+.fa-cogs
+{
```
Comment (Sebaxp95) [inline src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.css]: Please provide one bracket style in css

File: src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.html [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+    </div>
+  </mat-panel-title>
+  <mat-panel-description>
+    {{panelOpenState[i] ? 'less...' : 'more...'}}
+  </mat-panel-description>
+  </mat-expansion-panel-header>
+  <app-analysis-names-list
+      [AnalysisType]="type"
+      [AnalysisNames]="algorithms[type]"
+      ></app-analysis-names-list>
```
Comment (Sebaxp95) [inline src/app/analysis-types-list/analysis-types-list/analysis-types-list.component.html]: Move '>' line above

File: src/app/upload/upload.component.spec.ts [hunk 4]
```diff
@@ -24,7 +24,7 @@ import { MockBackend} from '@angular/http/testing';
 import { RouterTestingModule } from '@angular/router/testing';
 import { UploadComponent } from "./upload.component";
 import { UploadService } from "./shared/upload.service";
-import { FormsModule } from "@angular/forms";
+import {FormsModule, ReactiveFormsModule} from "@angular/forms";
```
Comment (Sebaxp95) [inline src/app/upload/upload.component.spec.ts]: All imports should be with spaces and with this: ' ' quotes
import { Module } from 'path'

**Pull request closed without merging**